spk/comskip/src/dsm-control.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# Package
+PACKAGE="comskip"
+DNAME="Comskip"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+
+
+case $1 in
+    start)
+        exit 0
+        ;;
+    stop)
+        exit 0
+        ;;
+    status)
+        exit 0
+        ;;
+    log)
+        echo "${INSTALL_DIR}/install.log"
```

Comment:
Nope ;) If Comskip doesn't maintain it's own log to link to from here, this can be removed.